R/plot_recruitment.R

```diff
@@ -163,5 +182,23 @@ plot_recruitment <- function(dat,
                     y = "Recruitment Deviations")
   }
   plt_fin <- add_theme(plt)
+
+  # add alt text and caption
+  plt_fin <- plt_fin +
+    ggplot2::labs(caption = caps_alttext[[1]],
```

Comment:
Where does this add in the caption and alt text? Is it just associated with the object or directly added to the plot? This needs to be tested in a quarto doc if referenceing stays the same as when using fig-cap: for quarto chunk ops

Reply:
The caption and alt text are added to the ggplot directly. asar will need to be updated so that these fxns work when integrated into it; I just focused on developing satf first. But here is an example change for asar, once we finalize satf's changes: we change create_template() so that there's an option for the user to indicate whether they already have rdas created for figures and tables.
If yes, then asar would directly import the rdas instead of making new figures/tables. Then, for each figure, we'd have two chunks: chunk 1 imports the rda, then chunk 2 plots the figure from the rda but removes the caption with `+ labs(caption = "")`. Then, it saves the fig.cap and fig.alt as the figure caption and alt text extracted from the rda (as mocked up in the screenshot in our 11/12 meeting notes)
If no, then asar would use similar infrastructure to what's currently present.
Does that help clarify the idea behind this choice?